src/System.Windows.Extensions/src/System/Drawing/FontConverter.cs

```diff
@@ -101,21 +103,16 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
 
         public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
         {
-            FontStyle f_style;
-            float f_size;
-            GraphicsUnit f_unit;
-            string font;
-            string units;
-            string[] fields;
-
             if (!(value is string))
             {
                 return base.ConvertFrom(context, culture, value);
             }
 
-            font = (string)value;
+            string font = (string)value;
```

Comment:
previously this would gracefully fall back to the base class if 'object' was not a string. 
https://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/FontConverter.cs,83
with this change, it will throw an exception.

Reply:
if the object is not a string then it will enter this if block https://github.com/dotnet/corefx/pull/39098/files#diff-0dccd18bdb029399c68857dc054ee0d7R106
am I missing something here ?

Reply:
A failed explicit cast like this will throw `System.InvalidCastException: 'Unable to cast object of type 'System.Object' to type 'System.String'.'`

To avoid that you can use the `is` operator but it is probably more efficient to use `as` followed by a null check, like the old code.